portal/server/routes/static.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/security.txt', (req, res) => {
```

Comment:
can we not use the `public` folder to server static files? we have other files in there as well

Reply:
We can't because public folder is configured to serve files from `/assets/` directory, which means our link will be `/assets/security.txt`.